src/styles/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(n + 1) {
```

Comment:
The `:nth-child(n + 1)` selector in `.stars--5` is not necessary since it targets all child elements. If you want all stars to be active in `.stars--5`, you should apply the `background-image` for the `.stars--5 .stars__star` directly.